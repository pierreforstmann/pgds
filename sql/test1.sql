--
-- test1.sql
create extension pgds;
--
create table t1(x int);
create table t2(x int);
--
select * from t1;
select * from t1;
--
insert into t2 values(2);
select * from t2;
select * from t2;
--
select * from t1;