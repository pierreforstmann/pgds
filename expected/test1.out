--
-- test1.sql
create extension pgds;
--
create table t1(x int);
create table t2(x int);
--
select * from t1;
INFO:  analyzing "public.t1"
INFO:  "t1": scanned 0 of 0 pages, containing 0 live rows and 0 dead rows; 0 rows in sample, 0 estimated total rows
 x 
---
(0 rows)

select * from t1;
INFO:  analyzing "public.t1"
INFO:  "t1": scanned 0 of 0 pages, containing 0 live rows and 0 dead rows; 0 rows in sample, 0 estimated total rows
 x 
---
(0 rows)

--
begin;
insert into t2 values(2);
INFO:  analyzing "public.t2"
INFO:  "t2": scanned 0 of 0 pages, containing 0 live rows and 0 dead rows; 0 rows in sample, 0 estimated total rows
select * from t2;
INFO:  analyzing "public.t2"
INFO:  "t2": scanned 1 of 1 pages, containing 1 live rows and 0 dead rows; 1 rows in sample, 1 estimated total rows
 x 
---
 2
(1 row)

select * from t2;
 x 
---
 2
(1 row)

commit;
select * from t2;
 x 
---
 2
(1 row)

--
select * from t1;
INFO:  analyzing "public.t1"
INFO:  "t1": scanned 0 of 0 pages, containing 0 live rows and 0 dead rows; 0 rows in sample, 0 estimated total rows
 x 
---
(0 rows)

